src/screens/LoginScreen.js

```diff
@@ -22,20 +34,66 @@ const styles = StyleSheet.create({
   },
   caption: {
     marginTop: 10,
+    marginRight: 12,
+    marginLeft: 12,
     textAlign: 'center',
     fontSize: 17,
     color: '#777',
   },
+  captionPhoneNumber: {
+    justifyContent: 'center',
+    display: 'flex',
+    textAlign: 'right',
+    marginTop: 10,
```

Comment:
Nit: need to increase space between caption and captionPhoneNumber, try increasing marginTop to 20 as the figma element suggests